draft.xml

```diff
@@ -244,11 +244,13 @@
             "2001:db8::1" and "2001:db8::/32" for IPv6.</t>
           </section>
 
-          <section title="Country">
-            <t>OPTIONAL. The country field, if non-empty, MUST be a 2 letter
+          <section title="Country / alpha2code">
```

Comment:
Suggest: title="Alpha2code [previously: country]"?

Reply:
Ugh, I dislike this, but sure, whatever...